### PR TITLE
Openrazer daemon cleanup patches

### PR DESCRIPTION
**Pull request by whot**
A bunch of patches to clean up the code a bit and make it easier to read. For the most part, this just moves things around into helper functions. The functionality changes are:
- assuming the root user is in plugdev
- ignoring screen saver monitor initialization failure

This is just the first set of patches, I suspect there will be more to come, but these are easy ones :)

### CODE REVIEW

**Comment by whot** [comment]
This includes two bits from #380 but not the actual 'bind to system bus' bits

File: daemon/openrazer_daemon/daemon.py [hunk 1]
```diff
@@ -194,22 +194,21 @@ def __init__(self, verbose=False, log_dir=None, console_log=False, run_dir=None,
...
+            ('razer.devices', 'getOffOnScreensaver', self.enable_turn_off_on_screensaver, 'b', None),
+            ('razer.devices', 'syncEffects', self.sync_effects, 'b', None),
+            ('razer.devices', 'getSyncEffects', self.get_sync_effects, None, 'b'),
+            ('razer.daemon', 'version', self.version, None, 's'),
+            ('razer.daemon', 'stop', self.stop, None, None),
+        }
+
+        for m in methods:
+            self.logger.debug("Adding {}.{} method to DBus".format(m[0], m[1]))
+            self.add_dbus_method(m[0], m[1], m[2], in_signature=m[3], out_signature=m[4])
```
Comment (z3ntu) [inline daemon/openrazer_daemon/daemon.py:226]: Not just easier to parse, also easier to read 😃

**Comment by terricain** [comment]
Looks good, will merge once I have proper internet :wink:

**Comment by whot** [comment]
fwiw, force-pushed an update, I had a copy/paste in the 'daemon: make the dbus method initialization easier to parse' commit

**Comment by whot** [comment]
And a few more updates, but I'll leave this branch alone now, except for any bugfixes I find :)

**Comment by terricain** [comment]
Havent tested it but it looks decent, resolve the conflicts and then its fine to merge.

**Comment by terricain** [comment]
Also I like #388 but i'd rather merge this first as its less to review :wink:

**Comment by whot** [comment]
conflicts resolved for this one and #388, should be good to go

**Comment by whot** [comment]
urgh, did you really intend to squash all these into a single giant patch? this would explain why #388 now has conflicts too

**Comment by terricain** [comment]
ahh good point

**Comment by whot** [comment]
which branch is the 2c032fd commit on? I can't find it anywhere

**Comment by terricain** [comment]
ahh I canceled that, I can revive it if you want?

**Comment by whot** [comment]
I think the best approach is to revert 39fdfc645c, push that to master and then re-merge this PR without squashing. That provides us with the history as it's supposed to be, and should make #388 mergeable too

**Comment by terricain** [comment]
@whot, are all these commits also contained in 388?

**Comment by whot** [comment]
yep, #388 is a fast forward on #385 so pulling that will give you 385 too.

**Comment by terricain** [comment]
@whot kl, will read through that one tomorrow evening, looks alright though

**Pull request merged**